src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts

```diff
@@ -35,6 +35,8 @@ export class ReadingsGraphComponent implements OnDestroy {
   public showSummarySpinner = true;
   public isSpectrum = false;
   public polyGraphData = {};
+  public showSummary = false;
+  public toggleSummaryGraphButtonText = 'Show Summary';
```

Comment:
it would be better to have `const SHOW_SUMMARY_TEXT =  'Show Summary'` and `const SHOW_GRAPH_TEXT =  'Show Graph' and use them to change at single place.